packages/common/src/dotlottie-player.ts

```diff
@@ -116,19 +135,22 @@ export class DotLottiePlayer {
     this._src = src;
     this._container = container;
 
-    this.setMode(options?.mode || PlayMode.Normal);
-
     if (options?.testId) {
       this._testId = options.testId;
     }
 
+    this._extraOptions = this._extractExtraOptions(options || ({} as DotLottieConfig<RendererType>));
+
+    this.setCount(this._extraOptions.count);
+    this.setIntermission(this._extraOptions.intermission);
+    this.setMode(this._extraOptions.mode);
```

Comment:
do we have runtime validation for these properties ?

Reply:
added validation to `_extractExtraOptions` function